Servers/middlewares/auth.middleware.ts

```
@@ -0,0 +1,28 @@
+import { NextFunction, Response } from "express";
+import { getTokenPayload } from "../utils/jwt.utils";
+import { getUsers } from "../utils/users.utils";
+import { IUserAuthRequest } from "../types/IUserAuthRequest";
+import { User } from "../models/user.model";
+
+const authenticateJWT = async (req: IUserAuthRequest, res: Response, next: NextFunction) => {
+    const token = req.headers.authorization && req.headers.authorization.split(" ")[1];
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)